src/index.html

```diff
@@ -15,8 +15,49 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="0"
```

Comment:
The `crossorigin` attribute is set to '0', which is not a valid value. It should be either 'anonymous' or 'use-credentials'. Please update this attribute to a valid value.